tests/framework/extensions/provisioning/creates.go

```diff
@@ -227,22 +227,30 @@ func CreateProvisioningCustomCluster(client *rancher.Client, externalNodeProvide
 		}
 	}
 
-	if clustersConfig.Hardened && clustersConfig.KubernetesVersion <= string(provisioninginput.HardenedKubeVersion) {
-		err = hardening.HardeningNodes(client, clustersConfig.Hardened, nodes, rolesPerNode)
-		if err != nil {
-			return nil, err
-		}
+	if clustersConfig.Hardened {
+		if strings.Contains(clustersConfig.KubernetesVersion, "k3s") {
```

Comment:
Nit; I think this can use `k3s` enum from `extensions/clusters/clustermeta`

Reply:
Can you elaborate on this? Not sure I follow?

Reply:
https://github.com/rancher/rancher/blob/release/v2.8/tests/framework/extensions/clusters/clustermeta.go#L22

Reply:
agreed, let's avoid doing raw string comparisons if we can.

Reply:
Addressed